rdm/0011-creators-contributors-metadata.md

```diff
@@ -0,0 +1,292 @@
+- RFC PR: [#11](https://github.com/inveniosoftware/rfcs/pull/11)
+- Authors: Tom Morrell, Guillaume Viger
+
+# Contributors / Creators Schema Definition
+
+## Summary
+
+This RFC defines the JSON schema field(s) holding the
+contributors, creators, authors... of a bibliographic record. Because defining
+the whole schema is a big effort, we have broken it off into constituent parts.
+
+
+## Motivation
+
+- As an InvenioRDM hoster, I want to benefit from a common and already defined
+  bibliographic schema, so that I don't have to make my own.
+- As an InvenioRDM hoster, I want to use a schema that is compatible with
+  DataCite schema (and others), so that I can inter-operate with other metadata
+  formats.
+- As an InvenioRDM developer, I want to code to a common metadata
+  schema, so that I don't repeat code and so that I benefit from and to others'
+  work.
+- As a researcher, I want to cite a record with its authors/creators,
+  so that my citation follows standards.
+- As a researcher, I want a DOI for my record which is only possible
+  if the equivalent `Creator` field is passed to the minting authority (DataCite).
+- As an InvenioRDM hoster, I want to provide a generic name field, so that
+  international authors and contributors can enter their full name in their own
+  way.
+- As an InvenioRDM hoster, I want to support person and organization identifiers,
+  so that I can disambiguate between entities.
+
+
+## Detailed design
+
+Creators and contributors can be people or organizations. Their schema structure
+will be reused. Their definition in the `"definitions"` section is:
+
+```json
+{
+  "definitions": {
+    "identifier": {
+      "type": "object",
+      "description": "A generic identifier.",
+      "additionalProperties": false,
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "Kind of identifier.",
+          "enum": [
+            "ads",
+            "ark",
+            "arxiv",
+            "bioproject",
+            "biosample",
+            "doi",
+            "ean13",
+            "ean8",
+            "ensembl",
+            "genome",
+            "gnd",
+            "hal",
+            "handle",
+            "isbn",
+            "isni",
+            "issn",
+            "istc",
+            "lsid",
+            "orcid",
+            "pmcid",
+            "pmid",
+            "purl",
+            "refseq",
+            "ror",
+            "sra",
+            "uniprot",
+            "url",
+            "urn"
+          ]
+        },
+        "value": {
+          "type": "string"
+        }
+      },
+      "required": ["type", "value"]
+    }
+  },
+  "person_or_org": {
+    "type": "object",
+    "description": "A person or organization.",
+    "additionalProperties": false,
+    "properties": {
+      "affiliation": {
+        "type": "string",
+        "description": "Affiliation for the purpose of the specific record."
+      },
```

Comment:
Went for a single affiliation as in Zenodo (versus multiple before)

Reply:
I'd lean toward supporting multiple affiliations, we have a few examples and it's annoying shoving them in one field.

Reply:
Just came across it when preparing the Open Repositories-Upload to zenodo:
```
 "authors": "Feger, Sebastian Stefan (1,2);(...)"
 "organisations": "1: CERN, Switzerland; 2: LMU Munich, Germany; 3: Humboldt-Universität zu Berlin, Germany"
```